roles/edpm_libvirt/templates/qemu.conf.j2

```diff
@@ -2,7 +2,6 @@ max_files = 32768
 max_processes = 131072
 vnc_tls = 0
```

Comment:
Do we have a jira for this?

Reply:
I don't think we have. Could you file one please?